views/index.ejs

```diff
@@ -23,54 +23,19 @@
     <!--Deployed commit: <%= process.env.HEAD_COMMIT %>-->
     <!--Build env: <%= process.env.NODE_ENV %>-->
 
-    <noscript>
-      <p style="text-align: center;">
-        Sorry, it looks like you have JavaScript disabled.
-        <strong><a href="http://enable-javascript.com/">Learn how to enable it.</a></strong>
-      </p>
-    </noscript>
+    <h1>Under Maintenance</h1>
+    
+    <p>The Zooniverse is currently under maintenance. We'll be up again in an hour.</p>
```

Comment:
I think maybe let's see if @lcjohnso and @mrniaboc have any input on the wording of this. I think we should probably apologise for the disruption.